doc/lsst.meas.algorithms/creating-a-reference-catalog.rst

```diff
@@ -155,3 +158,40 @@ You can query the ``refcats`` collection to see whether your htm shards appear:
     butler query-datasets --collections refcats REPO
 
 For LSST staff using ``lsst-devl``, see the `Reference catalogs policy <https://developer.lsst.io/services/datasets.html#reference-catalogs>`_ in the Developer Guide for additional policy about adding reference catalogs to the common repo.
+
+.. _lsst.meas.algorithms-refcat-existing:
+
+5. What to do with existing reference catalogs?
+===============================================
+
+Already existing reference catalogs (for example, the PS1 or Gaia DR2 catalogs that were used in gen2 butlers) can also be ingested into the butler, negating the need to convert them to the LSST format yourself.
```

Comment:
Maybe "... can be directly ingested into a gen3 repo as they are already in the LSST format."